### PR TITLE
fix squished profile image

### LINKED ISSUE #1204: Profile Image Shrink
**Describe the bug**
Profile Picture appears as  150px * 150px
so when it's aspect ration is not 1 the image will shrink 

**To Reproduce**
1. Go to 'https://circuitverse.org/users/18462/profile'

**Expected behavior**
there's two solution (i would go with the second solution): 
1 - user should crop his image to make it's aspect ratio to be 1 before submitting it 
2 - we can simply crop it on the front end by doing some css  

**Screenshots**
image skey

**Desktop (please complete the following information):**
 - OS: WIN10
 - Browser: chrome

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by mahmodHammad**
Fixes #1204

#### Describe the changes you have made in this PR -
- fix squished profile image
- make it rounded 

### After:
img fix2

### Before:
img bef

### CODE REVIEW

File: app/views/users/logix/index.html.erb [hunk 1]
```diff
@@ -29,10 +29,12 @@
     </div>
   <% end %>
 
+
   <div class ="d-flex flex-column align-items-center flex-md-row justify-content-md-between align-items-md-center">
     <div class="d-flex flex-row justify-content-center align-items-center">
-      <%= image_tag @user.profile_picture.url(:medium, alt: "@user.name"), style: "width: 100px; height: 100px; border: 0px;" %>
+      <%= image_tag @user.profile_picture.url(:medium, alt: "@user.name"), style: "width: 100px; height: 100px; border: 0px; margin: 10px; object-fit: cover; border-radius: 50%;" %>
```
Comment (tachyons) [inline app/views/users/logix/index.html.erb]: Move to stylesheet or use bootstrap classes to accomplish the same

File: app/views/users/logix/profile.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
       <div class="row profile-background">
         <div class="col-xs col-sm-12 col-md col-lg-6">
           <div class="row-about">
-            <%= image_tag @profile.profile_picture.url(:medium, alt: "@user.name"), style: "width: 150px; height: 150px; border: 0px; margin: 30px 30px 20px 30px;" %>
+            <%= image_tag @profile.profile_picture.url(:medium, alt: "@user.name"), style: "width: 150px; height: 150px; border: 0px; margin: 30px 30px 20px 30px; object-fit: cover; border-radius: 50%;" %>
```
Comment (tachyons) [inline app/views/users/logix/profile.html.erb]: Use bootstrap classes or move this to css

File: app/views/users/logix/index.html.erb (hunk 1 above)
Comment (mahmodHammad) [reply app/views/users/logix/index.html.erb]: Done

File: app/views/users/logix/profile.html.erb (hunk 2 above)
Comment (mahmodHammad) [reply app/views/users/logix/profile.html.erb]: Done

File: app/assets/stylesheets/logixRelease.scss [hunk 3]
```diff
@@ -91,6 +91,12 @@ a.btn.btn-outline-danger {
   margin-top: 40px;
 }
 
+.profile-image {
```
Comment (abstrekt) [inline app/assets/stylesheets/logixRelease.scss]: where is this class used? please remove them if unused.

Comment (mahmodHammad) [reply app/assets/stylesheets/logixRelease.scss]: Thank you brother ! ❤
- i some hove forgot to add this css class to image 🤦‍♀️ , but it's fixed now

**Pull request merged**